### DISCUSSION TITLE
How do I stop SumatraPDF from opening new windows and always using current?

### DISCUSSION THREAD

**Discussion by Luvirini** [sumatrapdfreader/sumatrapdf#4347] [Q&A]
About 1/3 of the time that I drag tabs around the tab opens in a new window instead of moving it . How do I stop this behavior as it seems something reasonably new bugged behavior?

**Comment by kjk** [comment]
You're dragging it out of tab area, it works as expected.

**Reply by Luvirini** [reply]
Ah, so the same answer as Firefox: If you have shaky hands, do not use our software as we like to discriminate? (At least there a extension exists that can fix the problem, but the Firefox team was like "Your problem that you have shaky hands")

So anyone know of what version is the last working one or if there is a workaround?

**Reply by GitHubRulesOK** [reply]
@Luvirini  I have shaky hands  too but the older one was worse as then you had to go back and drag again

I had reason to dig out my old tablet and every thing is smaller there

@kjk there are naturally more able bodied than disabled or else its a topsy turvey concept, However some of us will not get better and have to live with small icons etc. <strike>For those like me that need bigger menu it would be a god send to be able to make tabs bigger too</strike> FORGOT you can increase UI text height
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/54e8de2f-1824-4254-ab26-d3e04b769e68)

**Comment by gh2web** [comment]
@Luvirini The change was introduced in 3.5, so you want version: 3.4.6

Available at: https://www.sumatrapdfreader.org//dl//rel/3.4.6/SumatraPDF-3.4.6-64-install.exe

(I am sticking with that version for the same reason)
